### DISCUSSION TITLE
Implementar readmes semanales

### DISCUSSION THREAD

**Discussion by Noe3467** [CodeSystem2022/Perricornios-Tercer-Semestre#416] [Ideas]
Solo para utilizar este espacio y viendo el aporte de la scrum de esta semana. Está excelente que cada uno pueda (cuando le toque) implementar su metodología de trabajo y comunicarlo al equipo en cada carpeta de trabajo. Personalmente quedé encantada, no sé que les parece?
 
@NaSP32 @GabJaJ @bguzmanech @BrunoLCA95 @VillaC95 @asterisko19  @Noelia-Ruiz

**Comment by NaSP32** [comment]
Me pareció oportuno también dejar por aquí aclarada la fecha de entrega y
todas las opciones de comunicación que siempre proponemos por medios
informales. Estamos disponibles para que todos vayamos al mismo ritmo
acompañándonos.
